hack/remove-calico-policy/remove-calico-policy.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Set the clusterCIDR to the appropriate CIDR for this cluster.
+clusterCIDR=10.244.0.0/16
```

Comment:
We need to document this in the steps above.

Would be nice if this could be configured through the DaemonSet in an env var or something similar so folks don't need to modify the script.